File: packages/Pyodide/scripts/edit_df.js
//name: EditDFJS
//description: dataframe input/output
//language: javascript
//input: dataframe input_df
//output: dataframe output_df

const res = input_df.clone();
res.columns.addNewFloat('new_col');
for (let idx = 0; idx < res.rowCount; idx++) {
    res.set('new_col', idx, 1.0);
}
output_df = res;

File: packages/Pyodide/scripts/edit_df.py
#name: EditDFPy
#description: dataframe input/output
#language: pyodide
#input: dataframe input_df
#output: dataframe output_df

rowCount = len(input_df.index)
lst = [1] * rowCount
input_df['new_col'] = lst
output_df = input_df
